src/main/java/com/joshdev/deathmanager/libs/Serialization.kt

```diff
@@ -0,0 +1,44 @@
+/* Licensed under GNU General Public License v3.0 */
+package com.joshdev.deathmanager.libs
+
+import com.joshdev.deathmanager.DeathManager
+import org.bukkit.inventory.ItemStack
+import org.bukkit.inventory.PlayerInventory
+import org.json.simple.JSONArray
+import org.json.simple.JSONObject
+import org.json.simple.parser.JSONParser
+import java.util.Arrays
+
+class Serialization {
+    companion object {
+        fun Serialize(inv: PlayerInventory): String { // TODO Account for Armor and Shield slots.
```

Comment:
The Serialize and Deserialize methods should have opposite signature. The simplest test case for 99% of Serialization/Deserialization logic is:
```kotlin
val originalObject = [...] // Create some object you can serialize
val deserializedObject = Deserialize(Serialize(originalObject))
assert deserializedObject == originalObject
```
This should absolutely not care whether it serializes player inventory or a chest. That responsibility is higher up.

Reply:
In the Serialization class, Serialize() now takes List<ItemStack> as input and Deserialize outputs the same datatype, therefore this should now not care about what inventory it serializes (Hopefully!).

Reply:
Correction, not sure why I said List, I meant Array, sorry!